### DISCUSSION TITLE
how to get the lastest database schema data?

### DISCUSSION THREAD

**Discussion by jackshang** [jeremyevans/sequel#2151] [General]
db = Sequel.connect(....)

1. I add 2 colums for the table in derby db.

db.run("ALTER TABLE AIWO.TI_PAGE ADD TI_MODULE BIGINT")
db.run("ALTER TABLE AIWO.TI_PAGE ADD TI_FUNCTION BIGINT")

2. In fact, it's already added. Check all columns by the bellow snippet code, can not find the 2 new columns. 

db[:ti_page].columns

How to refresh the db schema data in Sequel?

**Comment by AlexWayfer** [comment]
Please, try `db.schema` method.

**Reply by jackshang** [reply]
thx

**Comment by jeremyevans** [comment]
You can use `columns!` instead of `columns` to force a reload.

**Reply by jackshang** [reply]
thx

**Reply by jackshang** [reply]
How about db.schema()? Is there have a method to force reload table's schema info?

**Reply by jackshang** [reply]
I found add a parameter can work well.
db.schema(:sys_object, :reload => true)
